build/cmake/init.cmake

```diff
@@ -36,6 +36,26 @@ if(MSVC)
               "Flags used by the CXX compiler during ${cfg_upper} builds." FORCE)
         endif()
     endforeach()
+    
+    if(wxUSE_OPTIMISE)
+        set(MVC_PREFERRED_RELEASE_FLAGS " /LTCG /OPT:REF,ICF /DEBUG")
+        wx_string_append(CMAKE_EXE_LINKER_FLAGS_RELEASE "${MVC_PREFERRED_RELEASE_FLAGS}")
+        wx_string_append(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${MVC_PREFERRED_RELEASE_FLAGS}")
```

Comment:
Visual Studio librarian really supports /LTCG (https://docs.microsoft.com/en-us/cpp/build/reference/ltcg-link-time-code-generation?view=vs-2019), but all described relates to linker... But it will not be worse if I add it